vendor/github.com/lacework/go-sdk/api/data_export_rules.go

```diff
@@ -0,0 +1,129 @@
+//
+// Author:: Darren Murray (<darren.murray@lacework.net>)
+// Copyright:: Copyright 2022, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+// DataExportRulesService is a service that interacts with the DataExportRule
+// endpoints from the Lacework Server
+type DataExportRulesService struct {
+	client *Client
+}
+
+type DataExportRulesResponse struct {
+	Data    []DataExportRule `json:"data"`
+	Message string           `json:"message"`
+}
+
+type DataExportRuleResponse struct {
+	Data    DataExportRule `json:"data"`
+	Message string         `json:"message"`
+}
+
+type DataExportRule struct {
+	ID     string               `json:"mcGuid,omitempty"`
+	Filter DataExportRuleFilter `json:"filters"`
+	Type   string               `json:"type"`
+	IDs    []string             `json:"intgGuidList"`
+}
+
+type DataExportRuleFilter struct {
+	Name            string   `json:"name"`
+	CreatedBy       string   `json:"createdOrUpdatedBy,omitempty"`
+	UpdatedTime     string   `json:"createdOrUpdatedTime,omitempty"`
+	Enabled         int      `json:"enabled"`
+	ProfileVersions []string `json:"profileVersions"`
+}
```

Comment:
You missed the `description` field from the schema:
```
        "description": {
          "description": "Summary of the data export rule.",
          "type": "string"
        },
```
```suggestion
	ProfileVersions []string `json:"profileVersions"`
	Description     string `json:"description"`
}
```

Reply:
Added